contents/blog/ccpa-compliant-product-analytics.md

```diff
@@ -0,0 +1,68 @@
+* The right to delete personal information collected from them (with some exceptions)
+* The right to opt-out of the sale of their personal information
+* The right to non-discrimination for exercising their CCPA rights.
+
+## What data is protected under CCPA?
+
+Only California residents have rights under the CCPA. A California resident is a person who resides in California, even if they are is temporarily outside of the state.
+
+The CCPA applies to for-profit businesses that meet any of the following:
+
+* Have a gross annual revenue of over $25 million
+* Buy, receive, or sell the personal information of 50,000 or more California residents, households, or devices
+* Derive 50% or more of their annual revenue from selling California residents’ personal information.
+
+Personal information is protected under CCPA and is considered information that identifies could be linked with a person.
+
+In short, any information which is tied to a specific individual can be considered Personal Information, from their social security number or license plate number to photos, emails, URLs, IP addresses or even pseudonyms.
+
+## What is the impact of CCPA on product analytics?
+
+The CCPA requires businesses to give consumers certain information in a “notice at collection”, this means when people sign-up to use your product you need to explain how you intend to use their data to improve the product for them.
+
+A notice at collection must list the categories of personal information businesses collect about consumers and the purposes for which they use the categories of information. The notice must also contain a link to the business’s privacy policy, where consumers can get more details on your privacy practices.
+
+The right to delete personal information is also covered by CCPA, you have 45 days to respond to a request from a user to delete any personal information stored about them.
+
+## How is CCPA compliance different with PostHog?
+
+PostHog is different from most other product analytics tools such as Mixpanel or Amplitude because it enables you to self-host on your own infrastructure and maintain full control of the data, this means you can decide on how and where to host any personal inforation. It also means you have full control of the underling database making it possible for you to easily share any information stored about an individual or delete any personal data. 
+
+## How to set PostHog up for GDPR compliant analytics
+
+### Step 1: Choose a hosting provider
+
+We recommend hosting PostHog on your own infrastructure, or a private cloud such as AWS, Google Cloud Platform or Microsoft Azure.
+
+### Step 2: Deploy PostHog
+
+Deploying PostHog onto your own infrastructure is very straightforward. You can follow our [standard deployment guides](https://posthog.com/docs/self-host) to get started, or [arrange a demo](https://posthog.com/book-a-demo) to see it in action first.
```

Comment:
Just a thought on the phrase "very straightforward". In light of recent feedback, are we still describing it in this way?